gtfs-realtime/spec/en/reference.md

```diff
@@ -140,7 +140,7 @@ Uncertainty applies equally to both time and delay. The uncertainty roughly spec
 Realtime update for arrival and/or departure events for a given stop on a trip. Please also refer to the general discussion of stop time updates in the [TripDescriptor](#message-tripdescriptor) and [trip updates entities](trip-updates.md) documentation.
 
 Updates can be supplied for both past and future events. The producer is allowed, although not required, to drop past events.
-The update is linked to a specific stop either through stop_sequence or stop_id, so one of these fields must necessarily be set.
+The update is linked to a specific stop either through stop_sequence or stop_id, so one of these fields must necessarily be set.  If the same stop_id is visited more than once in a trip, then stop_sequence must be provided in all StopTimeUpdates for that stop_id on that trip.
```

Comment:
Should there be a similar update to trip-updates.md line 18?

I might degrade 'must' to 'should' though. We've seen a number of producers for which it's just not feasible to keep stop_sequences synchronized with GTFS in the AVL system without some tricks.

Note also that stop_sequence is one of the most-prone-to-breaking identifiers across static feed versions (since they're often not pre-gapped for future-proofing), and GTFS-RT still lacks a means of indicating the target static feed version.